.pyinstaller/hooks/hook-sunpy.py

```diff
@@ -0,0 +1,14 @@
+from PyInstaller.utils.hooks import collect_data_files, collect_entry_point, collect_submodules, copy_metadata
+
+datas, hiddenimports = collect_entry_point("pytest11")
+
+datas += collect_data_files("sunpy")
+datas += collect_data_files("dask")
+datas += collect_data_files("drms")
+datas += copy_metadata("sunpy")
+
+hiddenimports += collect_submodules('sunpy')
+hiddenimports += collect_submodules('numpy.distutils')
+hiddenimports += collect_submodules('distutils')
+hiddenimports += ['skimage.filters.rank.core_cy_3d']
```

Comment:
Say what now? Why is this here?

Reply:
I encountered these missing imports when running the SunPy's tests for PyInstaller, adding them fixed it.

Reply:
I am curious as to how we got here in the import stack, I didn't realise we had even an optional transitive dependency on skimage.

Reply:
https://github.com/sunpy/sunpy/blob/f1c45df9f555ca34f5846ddc615e2c6bd5d25c0f/sunpy/image/coalignment.py#L24
Which leads to a hidden import in skimage is the reason.
This is essential I suppose so I don't think we need it just for the tests.